test/test_makefile.py

```
@@ -0,0 +1,32 @@
+import subprocess
+from pathlib import Path
+
+
+def test_xcop_find_prunes_excluded_directories(tmp_path):
+    (tmp_path / 'pom.xml').write_text('<project/>', encoding='utf-8')
+    for directory in ('.venv', '.env', 'wp'):
+        excluded = tmp_path / directory
+        excluded.mkdir()
+        (excluded / 'launcher manifest.xml').write_text('<xml/>', encoding='utf-8')
+
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)